src/Akka.TestKit.NUnit3.Tests/Bugfix44Spec.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Akka.Actor;
+using NUnit.Framework;
+
+namespace Akka.TestKit.NUnit3.Tests;
+
+[TestFixture]
+public class Bugfix44Spec : TestKit
+{
+    // need to have the DefaultConfig for the TestKit with this ActorSystem
+    public Bugfix44Spec() : base(ActorSystem.Create("Foo", TestKit.DefaultConfig))
+    {
+        
+    }
+    
+    [Test]
+    public void Should_use_provided_ActorSystem()
+    {
+        Assertions.AssertEqual("Foo",Sys.Name);
+    }
+    
+    [Test]
+    public void Should_use_provided_ActorSystem_again()
+    {
+        // Technically this will be a second, different ActorSystem but the names should be the same
+        Assertions.AssertEqual("Foo",Sys.Name);
```

Comment:
Perhaps a better assertion would be storing a static to verify it's different between the test runs?

``` csharp
private static ActorSystem previousSystem;

public Bugfix44Spec() : base(ActorSystem.Create("Foo", TestKit.DefaultConfig))
{
    previousSystem ??= Sys;
}

[Test]
// nit - 'again' in test name seems misleading here, making it sound like we're reusing the same instance
public void Should_use_provided_ActorSystem_again()
{
    Asserts.AssertNotEqual(previousSystem, Sys)
}
```

Reply:
Got it - I wanted to do that but wasn't sure how with NUnit. The `Order` parameter is what I needed. Thanks. I'll give that a shot.

Reply:
Done - test passes.